conf/services-example.yaml

```diff
@@ -106,4 +106,24 @@ account_password_hash: pbkdf2
 # account_nick_default_prefix
 # This option specifies a prefix to be put in front of a nickname forcibly
 # changed through AccountNickProtect. The default value is no prefix
-#account_nick_default_prefix: Guest
\ No newline at end of file
+#account_nick_default_prefix: Guest
+
+# account_nick_protect_restrict
+# This option prevents a user from executing any commands except those needed
+# to maintain their connection or identify while unidentified and using a
+# protected nick. PING, PONG, IDENTIFY, ID, NICK, and QUIT will be allowed unless
+# account_nick_protect_restricted_commands is set to different values. The default
+# value is false.
+#account_nick_protect_restrict: true
+
+# account_nick_protect_restricted_commands
+# This option allows whitelisted commands tobe specified for
+# account_nick_protect_restrict, mirroring the functionality of SHUN. We strongly
+# recommend not removing PING, PONG, ID, or IDENTIFY from this list.
```

Comment:
I would include "NICK" on the recommended list.

Reply:
I'm not sure this makes sense. A user should be able to change their nickname back to an unprotected nick, right? That's part of how the existing implementation works iirc.

Reply:
This list is commands they can execute, right? If someone goes to just this recommended list, the user would remain unable to change their nick, even back to the original one.

Reply:
Aah, right. Will fix